app/assets/javascripts/short-form/ShortFormDataService.js.coffee

```diff
@@ -407,17 +407,19 @@ ShortFormDataService = (ListingService) ->
           preferences[prefKey] = true
 
           if prefKey == 'rentBurden'
-            if file.rent_burden_type == 'lease'
-              preferences.documents.rentBurden[file.address].lease = {
-                proofOption: file.document_type
-                file: file
-              }
-            else
-              preferences.documents.rentBurden[file.address].rent[file.rent_burden_index] = {
-                id: file.rent_burden_index
-                proofOption: file.document_type
-                file: file
-              }
+            addressFiles = preferences.documents.rentBurden[file.address]
```

Comment:
@callaghanc I'm not sure this one works. Don't we still have to set the data on `preferences.documents.rentBurden[file.address]` rather than `addressFiles`?

Reply:
@creativecoder oh yep my bad, was a bit too much copy and paste from what i did in FileUploadService, will push a correction